syntax = "proto3";

message ImageMessage {
    bytes image = 1;
    string camera_name = 2;
    bool is_gray = 3;
    int32 height = 4;
    int32 width = 5;
    int32 id = 6;
}